src/http/caddy.rs

```diff
@@ -0,0 +1,131 @@
+use crate::utils::project_directory::get_rymfony_project_directory;
+
+use regex::Regex;
+use std::path::PathBuf;
+use std::fs;
+use std::process::Command;
+use std::process::Stdio;
+#[cfg(not(target_os="windows"))]
+use std::os::unix::fs::PermissionsExt;
+
+#[cfg(not(target_os="windows"))]
+use runas::Command as SudoCommand;
+
+const CADDY_VERSION_REGEX: &'static str = r"^v(2\.\d+\.\d+) ";
+
+#[cfg(target_os="windows")]
+const CADDY_BIN_FILE: &'static str = "caddy.exe";
+
+#[cfg(not(target_os="windows"))]
+const CADDY_BIN_FILE: &'static str = "caddy";
+
+pub(crate) const CADDYFILE: &'static str = "
+{
+    {{ debug }}debug
+    log {{ log_file }}
+    {{ use_tls }}local_certs
+}
+
+{{ host }}:{{ http_port }} {
+    root * {{ document_root }}
+
+    encode gzip
+
+    {{ use_tls }}tls internal
+    {{ with_server_sign }}header Server \"Rymfony\"
+    {{ without_server_sign }}header -Server
+
+    php_fastcgi 127.0.0.1:{{ php_port }} {
+        index {{ php_entrypoint_file }}
+        resolve_root_symlink
+    }
+
+    file_server browse
```

Comment:
Are you sure you want to enable `browse`? That means if a request is to a directory, users will get a directory listing instead. Probably not what you want.

Reply:
It was here mostly for debugging purposes, I indeed forgot to remove it, thanks for the good catch 👍